packages/core/test/Cardano/types/Strings/util.test.ts

```diff
@@ -0,0 +1,33 @@
+import { InvalidStringError, assertIsBech32WithPrefix, assertIsHexString } from '../../../../src/Cardano';
+
+describe('Cardano/types/Strings/util', () => {
+  test('assertIsBech32WithPrefix', () => {
```

Comment:
Best to break this down into more explicit test cases that describe each scenario.